test

```diff
@@ -80,27 +92,49 @@ function check_environment_started()
     fi
 )
 
+function run_isolated_tests()
+(
+  local harness="$1"
+  cd "${path_test}"
+
+  ws helm kubeval qa
+  rm -rf .my127ws/helm/qa/charts/ .my127ws/helm/qa/requirements.lock
```

Comment:
This is better off in the `ws helm kubeval qa` method so we can fix in project Jenkins pipelines too

Reply:
I find it better from a dev perspective keeping it, as I usually need to run it more than once. We are dealing with yaml and all as well as its templating :)

Reply:
Haha true, okay I'll move back to test and add a new shell command to Jenkinsfile.twig